src/java.base/share/classes/java/nio/channels/SelectionKey.java

```diff
@@ -338,9 +338,9 @@ public int interestOpsAnd(int ops) {
      * <p> An invocation of this method of the form {@code k.isReadable()}
      * behaves in exactly the same way as the expression
      *
-     * <blockquote><pre>{@code
+     * {@snippet lang=java :
      * k.readyOps() & OP_READ != 0
-     * }</pre></blockquote>
```

Comment:
The blockquote provided some indentation, that might make the example more readable.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/13870/commits/acd62d5ccadd0667b2b08aae9de531adbfdcc8b1.